tests/test_keyring.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+import subprocess
+from pathlib import Path
+
+
+# Update this pubkey ID if the key or its subkeys are updated; see README
+GPG_PUBKEY_ID = "gpg-pubkey-7b22e6a3-60966ad"
+
+# Basic acceptance testing for keyring package.
+# Expects fedora environment.
+
+RPM_QUERY_KEYRING_PACKAGE = ["rpm", "-q", "securedrop-workstation-keyring"]
```

Comment:
Unless these need to be reused, the tests would be easier to read if the commands were inlined where they get invoked.

Reply:
(reuse it a bit checking multiple different packages, but made it clearer what's happening at invocation)